Source/Core/DolphinQt2/Settings/InterfacePane.cpp

```diff
@@ -19,6 +20,56 @@
 
 #include "DolphinQt2/Settings.h"
 
+static QComboBox* MakeLanguageComboBox()
+{
+  constexpr struct
+  {
+    const wchar_t* name;
```

Comment:
DolphinWX uses wchar literals, though I'm not sure why. Maybe something to do with how WX interprets char* vs. wchar_t*?

Replacing these all with normal char* literals and QStringLiteral is documented as doing what we want, and seems to actually do so as well, so I've changed it to that.